components/emoji/emoji_list_item/emoji_list_item.tsx

```diff
@@ -2,43 +2,42 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-import PropTypes from 'prop-types';
-
 import Permissions from 'mattermost-redux/constants/permissions';
 import {Client4} from 'mattermost-redux/client';
+import {CustomEmoji} from 'mattermost-redux/types/emojis';
+import {ActionFunc} from 'mattermost-redux/types/actions';
 
 import DeleteEmoji from 'components/emoji/delete_emoji_modal';
 import AnyTeamPermissionGate from 'components/permissions_gates/any_team_permission_gate';
 
-export default class EmojiListItem extends React.PureComponent {
-    static propTypes = {
-        emoji: PropTypes.object.isRequired,
-        currentUserId: PropTypes.string.isRequired,
-        creatorDisplayName: PropTypes.string.isRequired,
-        creatorUsername: PropTypes.string,
-        currentTeam: PropTypes.object,
-        onDelete: PropTypes.func,
-        actions: PropTypes.shape({
-            deleteCustomEmoji: PropTypes.func.isRequired,
-        }).isRequired,
+export type Props = {
+    emoji: CustomEmoji;
+    emojiId?: string;
```

Comment:
I don't think this needs to be in the props here.

Reply:
It is used in index.ts and read this PR thread https://github.com/mattermost/mattermost-webapp/pull/6742

Reply:
Ahh, so it is. Thanks @Ayanrocks !